### DISCUSSION TITLE
关于开源版本默认组件库的疑问

### DISCUSSION THREAD

**Discussion by arcelan** [NetEase/tango#110] [Q&A] [answered]
看tango的milestone计划，【1.0 RC     : m2, 2024-02-01, 2024-04-30】
目前应该处理RC阶段

文档这里 https://netease.github.io/tango-site/docs/designer/customize/components 说【不提供私有的内置组件库】
因为目前tango默认的基础组件库还是比较少的，想问下以后也不会扩展默认的组件库么

**Comment by wwsun** [comment, marked as answer]
1. 是的，目前已经是 RC 版本了，只是版本号的规则没改
2. 目前内部的组件库暂时没有开源。社区版会在以 antd 为示例，进行扩展，部分已经提供了一些扩展示例。https://github.com/NetEase/tango-components
